packages/sdks/react-sdk/src/components/AuthProvider/AuthProvider.tsx

```diff
@@ -41,6 +41,12 @@ interface IAuthProviderProps {
   refreshCookieName?: string;
   // Function to get external token, for seamless migration from external system
   getExternalToken?: () => Promise<string>;
+  // Custom storage configuration for tokens and user data
```

Comment:
1. I would mention that the default is local storage 
2. I would not mention tokens and user data, there are stuff that may stored not for those two purposes, for devise for example. maybe something more generic?

```suggestion
  // Custom storage object for authentication related data 
```

Reply:
Done